eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/api/EclairApi.scala

```diff
@@ -53,13 +54,17 @@ trait EclairApi {
 
   def connect(nodeURI: NodeUri): Future[String]
 
+  def connect(nodeId: NodeId, host: String, port: Int): Future[String]
+
   def close(id: ChannelId, spk: ScriptPubKey): Future[String]
 
-  def findRoute(nodeId: NodeId): Future[Vector[NodeId]]
+  def findRoute(nodeId: NodeId, amountMsat: MilliSatoshis): Future[Vector[NodeId]]
+
+  def findRoute(invoice: LnInvoice, amountMsat: Option[MilliSatoshis]): Future[Vector[NodeId]]
 
-  def findRoute(invoice: LnInvoice): Future[Vector[NodeId]]
+  def forceClose(channelId: ChannelId): Future[String]
```

Comment:
Return type here would be better as `Future[DoubleSha256DigestBE]`

Reply:
Why?

Reply:
I believe it returns the TXID of the channel closing transaction

Reply:
For some reason it returns "ok": https://github.com/ACINQ/eclair/blob/master/eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala#L1622